node/utils/metrics/organization.ts

```diff
@@ -0,0 +1,96 @@
+import type { Logger } from '@vtex/api/lib/service/logger/logger'
+
+import type { Metric } from './metrics'
+import { sendMetric } from './metrics'
+
+interface UpdateOrganizationFieldsMetric {
+  update_details: { properties: string[] }
```

Comment:
Why use underscore_case here instead of camelCase like the rest? We usually only do this when we have no choice (e.g. a backend request that requires underscore_case on the body fields).

Does Redshift data usually use underscore? If so that's fine, just wanted to be sure.

Reply:
I found [this definition](https://internal-docs.vtex.com/Analytics/VTEX-Data-Platform/data-ingestion-API/?h=schemaless#schema-enforced-ingestion-adding-new-data-sources) for Redshift and previous [Mauro used snake_case too](https://github.com/vtex-apps/b2b-organizations-graphql/blob/master/node/utils/metrics/impersonate.ts#L6).